### PR TITLE
Prevent modification of admin and current user

### PR DESCRIPTION
**Pull request by nest1ing**
This commit prevents the tries to remove or change the role of the
current user and `admin`.

Signed-off-by: Alexander Filippov <a.filippov@yadro.com>

### CODE REVIEW

**Review by alatarum — DISMISSED** [review]

**Comment by AlexanderAmelkin** [comment]
> This commit prevents the tries to remove or change the role of the

`the tries` -> `attempts`

File: commands/user [hunk 1]
```diff
@@ -141,6 +141,16 @@ function cmd_delete {
     return 1
   fi
 
+  if [[ "${names[*]}" =~ 'admin' ]]; then
+    echo "Admin is builtin user and cannot be removed!" >&2
```
Comment (AlexanderAmelkin) [inline commands/user]: `is a built-in`

File: commands/user [hunk 2]
```diff
@@ -226,6 +236,16 @@ function cmd_set_role {
     return 1
   fi
 
+  if [[ "${name}" =~ 'admin' ]]; then
+    echo "Admin is builtin user and cannot be changed!" >&2
```
Comment (AlexanderAmelkin) [inline commands/user]: same here

**Review by AlexanderAmelkin — CHANGES_REQUESTED** [review]

File: commands/user (hunk 1 above)
Comment (nest1ing) [reply commands/user]: fixed

File: commands/user (hunk 2 above)
Comment (nest1ing) [reply commands/user]: fixed

**Comment by nest1ing** [comment]
> `the tries` -> `attempts`

Fixed

**Review by AlexanderAmelkin — APPROVED** [review]

**Pull request merged**